File: spring/src/main/java/de/stinner/anmeldetool/domain/models/NamiMember.java
package de.stinner.anmeldetool.domain.models;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class NamiMember {
    private Long memberId;
    private String firstname;
    private String lastname;

    private LocalDate dateOfBirth;

    private Rank rank;
    private Gender gender;
}

File: angular/src/app/user-management/nami-members/model/nami-member.ts
import { Gender } from 'src/app/lib/models/gender.enum';
import { Rank } from 'src/app/lib/models/rank.enum';

export interface NamiMember {
  memberId: number;
  firstname: string;
  lastname: string;
  rank: Rank;
  gender: Gender;
}
